### ISSUE TITLE
update starwars data

### ISSUE THREAD

**Issue by romainfrancois** [tidyverse/dplyr#5269]
Looks like swapi.co is no longer and swapi.dev is a replacement, and this includes data from SW7

**Comment by romainfrancois** [comment]
There seems to be some issues though, e.g. Luke is not "Human": https://github.com/Juriy/swapi/issues/5

**Comment by hadley** [comment]
Is there a need to update the data?

**Comment by romainfrancois** [comment]
Not necessarily, I've updated the links to avoid cran warnings, perhaps some day when this matures, we can add data from the 3rd trilogy, I'll just close now.
